lib/rom/rails/model/validator/uniqueness_validator.rb

```diff
@@ -66,8 +68,8 @@ def rom
         # @return [TrueClass,FalseClass]
         #
         # @api private
-        def unique?(name, value)
-          relation.unique?(name => value)
+        def unique?(name, value, scope)
+          relation.where(scope).unique?(name => value)
```

Comment:
Ah, cool! Yeah this is my first foray into rom so I'll have to keep in mind multiple adapters going forward.